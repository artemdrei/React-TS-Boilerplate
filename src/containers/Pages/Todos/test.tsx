import React from 'react';
import { screen } from '@testing-library/react';
import { testRender, makeTestStore } from '@root/tests/utils';

import Cmp from './';

describe('Todos Page', () => {
  beforeEach(() => {
    const store = makeTestStore();
    testRender(<Cmp />, store);
  });

  it('Should match snapshot', () => {
    const store = makeTestStore();
    const { container } = testRender(<Cmp />, store);

    expect(container).toMatchSnapshot();
  });

  it('Should has correct title', () => {
    expect(screen.getByText('List of Todos')).toBeTruthy();
  });
});
