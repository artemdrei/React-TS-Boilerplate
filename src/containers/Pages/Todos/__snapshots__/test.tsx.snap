// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todos Page Should match snapshot 1`] = `
<div>
  <h1>
    List of Todos
  </h1>
  <ul>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        delectus aut autem
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        quis ut nam facilis et officia qui
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        fugiat veniam minus
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        et porro tempora
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        Yes
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        laboriosam mollitia et enim quasi adipisci quia provident illum
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        qui ullam ratione quibusdam voluptatem quia omnis
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        illo expedita consequatur quia in
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        quo adipisci enim quam ut ab
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        Yes
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        molestiae perspiciatis ipsa
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        No
      </div>
    </li>
    <li>
      <h3>
        <span>
          Title: 
        </span>
        illo est ratione doloremque quia maiores aut
      </h3>
      <div>
        <span>
          Completed: 
        </span>
        Yes
      </div>
    </li>
  </ul>
</div>
`;
